tests/test_crossdock.py

```diff
@@ -1,13 +1,13 @@
 # Copyright (c) 2016-2018 Uber Technologies, Inc.
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
+# Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
Please revert this

Reply:
please do not change license headers, they are standard and not subject to code guidelines

Reply:
okay, I'll revert back the changes..